src/_data/b2b/toc/marketing.yml

```diff
@@ -70,8 +70,184 @@ pages:
           url: "/sales-channels/google-ads/ads-edit.html"
         - label: Campaign Reporting
           url: "/sales-channels/google-ads/google-reporting.html"
-    - label: Amazon Sales Channel
-      url: "/sales-channels/amazon/amazon-sales-channel.html"
+    - label: Amazon Sales Channel 4.0+
+      url: "/sales-channels/asc/amazon-sales-channel.html"
+      children:
+      - label: Amazon Sales Channel
+        url: "/sales-channels/asc/amazon-sales-channel.html"
```

Comment:
because the child page use to be the parent page.  without creating individual landing pages for 4.0+ and 2x/3x, there is nothing to link the new parent labels to